cypress/support/commands.ts

```diff
@@ -30,7 +48,9 @@ Cypress.Commands.overwrite('tab', (originalFn, subject) => {
 
   log.snapshot('before', {next: 'after'});
 
-  return Cypress.Promise.try(() => originalFn(subject))
+  return Cypress.Promise.try(() => {
+    return originalFn(subject, options);
```

Comment:
This fixes an issue where `cy.tab({ shift: true })` didn't work, because the override didn't pass `options`